Gems/EMotionFX/Code/Source/Editor/ColliderContainerWidget.cpp

```diff
@@ -568,6 +484,18 @@ namespace EMotionFX
             m_colliderWidgets[i]->hide();
             m_colliderWidgets[i]->Update(nullptr, nullptr, InvalidIndex, PhysicsSetup::ColliderConfigType::Unknown, AzPhysics::ShapeColliderPair());
         }
+
+        for (ColliderWidget* colliderWidget : m_colliderWidgets)
```

Comment:
Are those coded needed? The visibility of colliderWidget should already been taken care of in the colliderWidget's update function, no? The owner widget shouldn't be controlling the visibility of the children widget, especially if the children widget itself already control the visibility itself.